integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -84,6 +90,12 @@ struct TestReshardingParameters {
     chunk_ranges_to_drop: HashMap<ShardIndex, std::ops::Range<i64>>,
     shuffle_shard_assignment_for_chunk_producers: bool,
     track_all_shards: bool,
+    // Manually specify what shards will be tracked for a given client ID.
+    // The client ID must not be used for any other role (validator, RPC, etc.).
+    // The schedule length must be at least `TESTLOOP_NUM_EPOCHS_TO_WAIT` so that it covers all epoch heights used in the test.
```

Comment:
should the length be greater then, as shown in the assertion on line 185?

Reply:
fixed in 66f279e879863ae6b1144cdd6ba70c595c62cf6c